packages/source-iotsitewise/src/time-series-data/client/withinLatestPropertyDataThreshold.ts

```diff
@@ -0,0 +1,11 @@
+import { SECOND_IN_MS } from '@iot-app-kit/core';
+
+export const RAW_DATA_RECENCY_THRESHOLD = 10; // in seconds
```

Comment:
may be ? ` RAW_DATA_RECENCY_THRESHOLD_SECS` .. its better to have the variable say the unit